### ISSUE TITLE
Uncaught Error in <ReactPlayer> invalid hook

### ISSUE THREAD

**Issue by ridhwaans** [Lucasmg37/react-netflix-player#38]
hello,
I upgraded react to the latest version and now there is an issue rendering the ReactPlayer component. Didnt happen before

```
react-dom.development.js:18687 The above error occurred in one of your React components:
...
line 115341
line 115342    e.default = function (n) {
line 115343        var e = n.title,
line 115344             t = void 0 !== e && e,
line 115345             a = n.subTitle,
    ...
    at e.default (http://localhost:3000/static/js/bundle.js:115343:15)
    at div
    at Player (http://localhost:3000/static/js/bundle.js:8314:108)
    at div
    at div
    at Tab (http://localhost:3000/static/js/bundle.js:6132:104)
    at Routes (http://localhost:3000/static/js/bundle.js:122353:5)
    at Router (http://localhost:3000/static/js/bundle.js:122286:15)
    at BrowserRouter (http://localhost:3000/static/js/bundle.js:121095:5)
    at SWRConfig$1 (http://localhost:3000/static/js/bundle.js:137530:21)
    at App (http://localhost:3000/static/js/bundle.js:2528:86)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.

react-dom.development.js:12056  Uncaught Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.
    at resolveDispatcher (react.development.js:1476:1)
    at useRef (react.development.js:1515:1)
    at e.default (ReactNetflixPlayer.js:1:1)
    at renderWithHooks (react-dom.development.js:16305:1)
    at mountIndeterminateComponent (react-dom.development.js:20074:1)
    at beginWork (react-dom.development.js:21587:1)
    at beginWork$1 (react-dom.development.js:27426:1)
    at performUnitOfWork (react-dom.development.js:26557:1)
    at workLoopSync (react-dom.development.js:26466:1)
    at renderRootSync (react-dom.development.js:26434:1)
  ```
here is the package.json
```
"dependencies": {
     "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-netflix-player": "^1.1.7",
    "react-redux": "^8.0.2",
    "react-router-dom": "^6.3.0",
    "react-scripts": "^5.0.1",
    "redux": "^4.2.0",
},
```
Ran the same [example ](https://github.com/Lucasmg37/react-netflix-player/blob/master/example/index.js)

```
import React, { useEffect, useContext } from 'react';
import ReactNetflixPlayer from "react-netflix-player";

const Player = () => {
const [playerItem, setPlayerItem] = useSharedState('playerContext')

 return (
      <React.Fragment>
      {playerItem && (
        <div id={"player"} >
          <ReactNetflixPlayer
          ...
          />
        </div>
        )}
        </React.Fragment>
    );
}
export default Player
```

do you have any clue why this is happening? thank you

**Comment by ridhwaans** [comment]
Doesnt work in React 18 because of peer dependency https://github.com/Lucasmg37/react-netflix-player/blob/dca0683688cfde812999074c8db1180dc7f0ce51/package.json#L52

PR: https://github.com/Lucasmg37/react-netflix-player/pull/39

**Comment by Lucasmg37** [comment]
PR #39 merged and pusblished at version 1.1.8 👍.
Thanks a lot @ridhwaans
